Web Components is breaking Blazor build

## Description  
latest Blazor release build failed in buildBlazorWebComponents with the following error:

##[error][webpack-cli] ModuleNotFoundError: Module not found: Error: Can't resolve './themes/button' in 'D:\Agent\2\_work\13\s\XPlatform\Main\Source\Infragistics.Blazor\src\ig\igniteui-webcomponents\components\button-group'
23:17:04 [ERR] [webpack-cli] ModuleNotFoundError: Module not found: Error: Can't resolve './themes/button' in 'D:\Agent\2\_work\13\s\XPlatform\Main\Source\Infragistics.Blazor\src\ig\igniteui-webcomponents\components\button-group'
##[error]Did you mean 'button.js'?
23:17:04 [ERR] Did you mean 'button.js'?
##[error]BREAKING CHANGE: The request './themes/button' failed to resolve only because it was resolved as fully specified
23:17:04 [ERR] BREAKING CHANGE: The request './themes/button' failed to resolve only because it was resolved as fully specified
##[error](probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
23:17:04 [ERR] (probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
##[error]The extension in the request is mandatory for it to be fully specified.
23:17:04 [ERR] The extension in the request is mandatory for it to be fully specified.
##[error]Add the extension to the request.


In addition to fixing this issue. A check should be added upstream to prevent these kinds of issues from breaking the Blazor build as this appears to happen often.

--- a/src/components/button-group/toggle-button.ts
+++ b/src/components/button-group/toggle-button.ts
@@ -1,8 +1,8 @@
 import { LitElement, html } from 'lit';
 import { property, query } from 'lit/decorators.js';
 
-import { all } from './themes/button.js';
 import { styles } from './themes/button.base.css.js';
+import { all } from './themes/button.js';
 import { themes } from '../../theming/theming-decorator.js';
 import { alternateName } from '../common/decorators/alternateName.js';
 import { registerComponent } from '../common/definitions/register.js';

--- a/src/components/button-group/toggle-button.ts
+++ b/src/components/button-group/toggle-button.ts
@@ -1,7 +1,7 @@
 import { LitElement, html } from 'lit';
 import { property, query } from 'lit/decorators.js';
 
-import { all } from './themes/button';
+import { all } from './themes/button.js';
 import { styles } from './themes/button.base.css.js';
 import { themes } from '../../theming/theming-decorator.js';
 import { alternateName } from '../common/decorators/alternateName.js';

